roles/olm/tasks/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# do any asserts here
+
+- include_tasks: install.yaml
+  when: operator_lifecycle_manager_install | bool
+
+- include_tasks: remove.yaml
+  when: operator_lifecycle_manager_remove | bool
```

Comment:
The pattern I've seen is setting one variable for this (instead using "when: not operator_lifecycle_manager_install").

Reply:
I'll follow up with that in the future - I want to make sure everything is installing correctly first, if that's okay